transform-plugins/src/main/java/co/cask/hydrator/plugin/XMLToJSON.java

```diff
@@ -133,6 +134,7 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
 
     @Name("outputField")
     @Description("The field containing the XML string to convert into a JSON string.")
+    @Macro
```

Comment:
Same as above, inputField is used for schema validation, which can't be deferred to `prepareRun` as of now.